featuretools/tests/primitive_tests/test_identity_features.py

```diff
@@ -11,6 +11,7 @@ def test_serialization(es):
     value = ft.IdentityFeature(es['log']['value'])
 
     dictionary = {
+        'name': None,
```

Comment:
So I suppose that with this new PR, old features generated using the previous feature serializer will now break because they don't contain the name field, right?
Would it be possible to make this backwards compatible? Could we do a `argments.get('name', None)` at this line
https://github.com/Featuretools/featuretools/pull/576/files#diff-a6f70d99f6dca96d42267829c30c764cR346

Reply:
There is now `SCHEMA_VERSION` (in `features_serializer.py`) which should be updated when the serialization format changes.